airbyte-integrations/connectors/source-smartsheets-2/acceptance-test-config.yml

```diff
@@ -0,0 +1,39 @@
+# See [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-smartsheets-2:dev
```

Comment:
Name is not set in stone. I saw a `source-mongodb-2` type of connector in there. I think it has the `2` because it is for MongoDB version 2, so naming this `smartsheets-2` is not exactly equivalent. But it should not be too hard to update the name in the future. For conventions, I would imagine that the Airbyte maintainers would not let us have two connectors for the same thing.